ipfs/evolution.md

```diff
@@ -1 +1,143 @@
-# IPFS - Evolution
\ No newline at end of file
+# IPFS - Evolution
+
+The IPFS project was born in 2013 by Juan Benet,  Juan  known Git's data-linking structure is useful across many data types–not just code. Juan recognized that combining this concept with the P2P file-sharing structure of BitTorrent would be powerful: functional, secure information sharing without centralized barriers. This could transform the world far beyond scientific datasets.
+
+The IPFS whitepaper was published in july 2014 and it caught the attention of many enthusiastis people. The contributors worked nigth, weekends and initally for free because they believed in the positive impact that open networks like IPFS could have on the world. The next year IPFS team was growing to get a coworking space in Seattle to implement and release the go-ipfs in April 2016. The improvements of 0.4.0 transitioned IPFS from an "exciting demo" to a genuinely useful tool for early adopters. In 2018 IPFS entered the next phases of its maturation. The project need to deliver on its ideological ans technical advantages at scale. IPFS has come a long way in the journey to building a faster, safer, and more open web to preserve and grow humanity's knowledge.
+
+The main difference in the evolution of the first release and the newest are the speed to adding a file, providing,  finding and fetching files. In 2016 IPFS got the first release with a few things comparing now, in this case the important improves are: the security of files, the usability of IPFS and the control of the network system.
+
+## Lines of code added and removed
+Commits
+![Commits](assets/commits.png)
+
+Aditions
+![Commits](assets/aditions.png)
+
+Deletions
+![Commits](assets/deletions.png)
+
+Code Frequency
+![Commits](assts/../assets/code_frec.png)
+
+
+
+
+# ADR1: Implement QUIC - Release 0.4.18
+
+## Context
+
+- QUIC is a new UDP-based network transport, can share a single file descriptor between all connections. This release has introduced experimental support for the QUIC protocol.
+
+## Decision
+
+- Implement QUIC as a new protocol to solve issues with TCP.
+
+
+## Status
+
+- Implemented
+
+## Consequences
+
+- Fewer local resources: allow us to dial faster and keep more connections open.
+- Faster connection establishment: significantly reduce the latency of DHT queries.
+- Behaves better on lossy networks: QUIC handles multiplexing internally, dropping a single packets affects only the related stream.
+
+
+# ADR2: Performance Resources - Release 0.4.18
+
+## Context
+
+- Actual the resource of utlization are a problem when it use the IPFS becouse the use a lot of CPU and memory.
+
+## Decision
+
+- Changed two of our most frequently used datastructures, CIDs and Multiaddrs, to reduce allocation load.
```

Comment:
Its clear that you copied this from somewhere else (which is fine) but at least change the tense of the verb ("**our** most frequently used datastructures")

Reply:
What are CIDs and Multiaddrs